### PR TITLE
Add NoonScan

### LINKED ISSUE #4135: NoonScan
### Source name

NoonScan 0.1

### Source link

https://noonscan.com/manga/?order=update

### Source language

Arabic

### Other details

_No response_

### Acknowledgements

- [X] I have checked that the extension does not already exist by searching the [extension listing](https://keiyoushi.github.io/extensions/) and verified it does not appear there.
- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a meaningful title with the source name.
- [X] I will fill out all of the requested information in this form.

### PR DESCRIPTION
**Pull request by vetleledaal**
Closes #4135

Checklist:

- [x] Updated `extVersionCode` value in `build.gradle` for individual extensions
- [x] Updated `overrideVersionCode` or `baseVersionCode` as needed for all multisrc extensions
- [x] Referenced all related issues in the PR body (e.g. "Closes #xyz")
- [x] Added the `isNsfw = true` flag in `build.gradle` when appropriate
- [x] Have not changed source names
- [x] Have explicitly kept the `id` if a source's name or language were changed
- [x] Have tested the modifications by compiling and running the extension through Android Studio
- [x] Have removed `web_hi_res_512.png` when adding a new extension

### CODE REVIEW

File: src/ar/noonscan/src/eu/kanade/tachiyomi/extension/ar/noonscan/NoonScan.kt [hunk 1]
```diff
@@ -0,0 +1,37 @@
+package eu.kanade.tachiyomi.extension.ar.noonscan
+
+import eu.kanade.tachiyomi.multisrc.mangathemesia.MangaThemesia
+import java.text.SimpleDateFormat
+import java.util.Locale
+
+class NoonScan : MangaThemesia(
+    "نون سكان",
```
Comment (vetleledaal) [inline src/ar/noonscan/src/eu/kanade/tachiyomi/extension/ar/noonscan/NoonScan.kt:8]: Not sure if this should be just:
```suggestion
    "NoonScan",
```

File: src/ar/noonscan/src/eu/kanade/tachiyomi/extension/ar/noonscan/NoonScan.kt [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    // override val pageSelector = "div#readerarea img" // 9
+    // override val sendViewCount: Boolean = true // 5
+    // override class SelectFilter(...) = ...
+    // override val statusOptions = arrayOf(...) // 0
+    // override val typeFilterOptions = arrayOf(...) // 0
+    // override val orderByFilterOptions = arrayOf(...) // 2
+    // override val projectFilterOptions = arrayOf(...) // 0
+    // override fun getGenreList(): List<Genre> = ... // 16
+    // override val hasProjectPage = false // 41
+    // override fun Element.imgAttr(): String = when Ellipsis // 1
```
Comment (AwkwardPeak7) [inline src/ar/noonscan/src/eu/kanade/tachiyomi/extension/ar/noonscan/NoonScan.kt]: remove?

File: src/ar/noonscan/src/eu/kanade/tachiyomi/extension/ar/noonscan/NoonScan.kt (hunk 1 above)
Comment (AwkwardPeak7) [reply src/ar/noonscan/src/eu/kanade/tachiyomi/extension/ar/noonscan/NoonScan.kt:8]: arabic is fine

File: src/ar/noonscan/src/eu/kanade/tachiyomi/extension/ar/noonscan/NoonScan.kt (hunk 2 above)
Comment (vetleledaal) [reply src/ar/noonscan/src/eu/kanade/tachiyomi/extension/ar/noonscan/NoonScan.kt]: oops

**Review by AwkwardPeak7 — APPROVED** [review]

**Pull request merged**